taxi/views.py

```diff
@@ -87,3 +88,47 @@
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(View):
+    def get(self, request):
+        form = DriverCreationForm()
+        return render(request,
+                      "taxi/driver_list.html", {"form": form})
+
+    def post(self, request):
+        form = DriverCreationForm(request.POST)
+        if form.is_valid():
+            form.save()
+            return redirect("taxi:driver-list")
+        return render(request,
+                      "taxi/driver_list.html", {"form": form})
```

Comment:
Similar to the `get` method, ensure that the `taxi/driver_list.html` template is suitable for displaying form errors when the form is invalid. Consider using a dedicated form template if necessary.